scripts/irods/test/resource_suite.py

```diff
@@ -36,15 +36,19 @@ def setUp(self):
         print("run_resource_setup - BEGIN")
         self.testfile = "testfile.txt"
         self.testdir = "testdir"
-        self.testresc = "TestResc"
-        self.anotherresc = "AnotherResc"
 
         hostname = lib.get_hostname()
         hostuser = getpass.getuser()
+
+        self.testresc = "TestResc"
+        self.testvault = "/tmp/" + hostuser + "/" + self.testresc
+        self.anotherresc = "AnotherResc"
+        self.anothervault = "/tmp/" + hostuser + "/" + self.anotherresc
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Probable insecure usage of temp file/directory.](https://app.codacy.com/app/trel/irods/pullRequest?prid=4593032)